sympy/integrals/transforms.py

```diff
@@ -1885,14 +1966,25 @@ def laplace_transform(f, t, s, legacy_matrix=True, **hints):
     Examples
     ========
 
-    >>> from sympy import DiracDelta, exp, laplace_transform
-    >>> from sympy.abc import t, s, a
+    >>> from sympy import DiracDelta, exp, laplace_transform, diff, Function, symbols
+    >>> y = Function('y')
+    >>> Y = Function('Y')
+    >>> z = Function("z")
+    >>> Z = Function("Z")
+    >>> s, t, a, b, c, d, z0, z1, y0 = symbols('s, t, a, b, c, d, z0, z1, y0')
     >>> laplace_transform(t**4, t, s)
     (24/s**5, 0, True)
     >>> laplace_transform(t**a, t, s)
     (gamma(a + 1)/(s*s**a), 0, re(a) > -1)
     >>> laplace_transform(DiracDelta(t)-a*exp(-a*t),t,s)
     (s/(a + s), Max(0, -a), True)
+    >>> laplace_transform(diff(z(t), t, 2), t, s)
+    s**2*LaplaceTransform(z(t), t, s) - s*z(0) - Subs(Derivative(z(t), t), t, 0)
+    >>> laplace_transform([diff(z(t), t, 2)-a*z(t)-b*y(t), diff(y(t), t)-c*z(t)-d*y(t)], t, s, {z(t): (Z(s), [z0, z1, 0]), y(t): (Y(s), [y0, 0, 0])})
+    [-a*Z(s) - b*Y(s) + s**2*Z(s) - s*z0 - z1, -c*Z(s) - d*Y(s) + s*Y(s) - y0]
+    >>> laplace_transform([diff(z(t), t, 2)-a*z(t)-b*y(t), diff(y(t), t)-c*z(t)-d*y(t)], t, s, {z(t): (Z(s), 0), y(t): (Y(s), 0)})
```

Comment:
These lines are too long. It would be better to split this up e.g. by first defining the odes:
```
odes = [diff(z(t), t, 2)-a*z(t)-b*y(t), diff(y(t), t)-c*z(t)-d*y(t)]
```

Also I think that it would be better to visually separate these from the simpler examples of using `laplace_transform` shown above by breaking it up like:
```
>>> test1
out1

It is also to compute Laplace transforms of functions that involve undefined functions and their derivatives which can be used for working with ordinary differential equations.

>>> more test
more out
```

Reply:
Good idea, thanks!